pkg/manifest/environment_test.go

```diff
@@ -38,21 +40,23 @@ func TestEnv(t *testing.T) {
 }
 
 func fakeGitopsDir(t *testing.T) (string, func()) {
-	tmpDir, cleanUp := makeTempDir(t)
+	appFS := afero.NewOsFs()
+
+	tmpDir, cleanUp := makeTempDir(t, appFS)
 	gitopsDir := filepath.Join(tmpDir, "gitops")
-	err := os.Mkdir(gitopsDir, 0755)
+	err := appFS.Mkdir(gitopsDir, 0755)
 	if err != nil {
 		t.Fatalf("failed to create gitops directory")
 	}
 	return gitopsDir, cleanUp
 }
 
-func makeTempDir(t *testing.T) (string, func()) {
+func makeTempDir(t *testing.T, f afero.Fs) (string, func()) {
 	t.Helper()
 	dir, err := ioutil.TempDir(os.TempDir(), "test")
```

Comment:
Is ioutil.TempDir() an "os" filesystem call?

Reply:
Using the afero.Fs should allow us to eliminate the use of this `makeTempDir` call.